index.html

```diff
@@ -209,47 +248,60 @@ <h2 class="title"> Skills</h2>
 
     </section>
 
+    <!-- Contact Me Section -->
+
     <section class="contact">
-        <article class="art-contact">
-            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-        </article>
-
-        <form action="https://formspree.io/f/mgebgoge" method="POST">
-            <input type="text" 
-            id="user_name"
-            name="full_name" 
-            placeholder="Full Name"
-            maxlength="30"
-            required
-            >
-
-            <input type="email" 
-            id="user_email"
-            name="email"
-            maxlength="30" 
-            placeholder="Email address"
-            required>
+
+        <div class="contact-container">
+
+            <div  class="contact-container-2">
+
+                <article class="art-contact">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </article>
```

Comment:
please kindly increase the font size of the text in the contact section `I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.`